protocol/avdl/metadata.avdl

```diff
@@ -33,4 +39,12 @@ protocol metadata {
   bytes getFolderHandle(string folderID, string signature, string challenge);
   void getFoldersForRekey(KID deviceKID);
   void ping();
+
+  // Merkle tree specific commands
+  // todo: we probably also want a method which will return a minmum set of interior nodes
+  // and a leaf node given a root node and a TLF ID.
+  MerkleRoot getMerkleRoot(string tree, long seqNo);
+  MerkleRoot getMerkleRootLatest(string tree);
+  MerkleRoot getMerkleRootSince(string tree, long when);
+  bytes getMerkleNode(string hash);
```

Comment:
It seemed consistent with other things like folder IDs and KIDs. It's hex encoded. I could switch it to bytes. No strong preference.